docs/admin/workspace-proxies.md

```diff
@@ -147,6 +147,27 @@ and up the deployment's replicas.
 coder wsproxy server
 ```
 
+### Running as a system service
+
+If you've installed Coder via a [system package](../install/index.md) Coder, you
```

Comment:
```suggestion
If you've installed Coder via a [system package](../install/index.md), you
```

Reply:
duplication

Reply:
copy and pasta. let me fix the source too..